Runtimes/Core/CMakeLists.txt

```diff
@@ -103,6 +103,13 @@ option(SwiftCore_ENABLE_FILESYSTEM_SUPPORT "Build for systems that have a filesy
 option(SwiftCore_ENABLE_OS_TRACE_LAZY_INIT "Use os_trace call to check if lazy init has been completed before making os_signpost calls." ${SwiftCore_HAS_OS_TRACE})
 option(SwiftCore_HAS_DARWIN_LIBMALLOC "Use Darwin malloc features" ${APPLE})
 
+if("${APPLE}" OR "${LINUX}")
```

Comment:
This is what `cmake/modules/DefaultSettings.cmake` is for.
`"${APPLE}"` should be `APPLE` and `"${LINUX}"` should be `LINUX`, but the `ON` cases can also just go under the appropriate platforms in `DefaultSettings.cmake`, and then use `default_set(SwiftCore_ENABLE_PRESPECIALIZATION "Enable generic metadata prespecialization")` from this file.

Reply:
Thanks for the feedback, and sorry for not reading the code more thoroughly before hand -- this is addressed in f757406517c8d5fa895f5870147b4570c7d512d6